subsys/logging/log_output_syst.c

```diff
@@ -17,6 +17,8 @@
 static struct mipi_syst_header log_syst_header;
 static struct mipi_syst_handle log_syst_handle;
 
+#define DATA_ID 0x1A
```

Comment:
Can you add some comment about what this value means ?

Reply:
This is the data id which is a numeric value in the range of 0..31 to identify the content pointed to by data. I have removed the macro for now to be consistent with how MIPI_SYST_WRITE was used in v1 logging implementation.